vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -80,7 +83,7 @@
  * @see <a href="https://developer.vimeo.com/api">The Vimeo API Docs</a>
  */
 @SuppressWarnings("unused")
-public class VimeoClient {
+final public class VimeoClient {
```

Comment:
In another PR I'd suggest some refactoring here:
- Move Retrofit setup code to another object.
- If we don't care about PUT/POST/DELETE response objects we could maybe return `Call<Boolean>` instead of returning `Call` or `Call<Object>`.
- Move cache & corresponding util methods into it's own class.
- Move auth logic into it's own class.
- This object has very low cohesion (it's responsible too much).  We could either move bits of logic into other classes (eg call the PinCode logic stuff), or we could spit each logical endpoint into it's own class and have this class be a bunch of single-line methods.

General questions:
- Some objects return a `Call<T>` and some take a `VimeoCallback`.  Is there a reason we don't stick with 1 over the other?